Microsoft.jQuery.Unobtrusive.Validation.nuspec

```diff
@@ -0,0 +1,29 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.jQuery.Unobtrusive.Validation</id>
+    <version>3.2.6</version>
+    <title>Microsoft jQuery Unobtrusive Validation</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft,aspnet</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <licenseUrl>http://www.microsoft.com/web/webpi/eula/net_library_eula_ENU.htm</licenseUrl>
```

Comment:
Same set of comments from https://github.com/aspnet/jquery-ajax-unobtrusive/pull/31 apply here.